internal/tunnel/argo.go

```diff
@@ -70,7 +76,7 @@ func NewArgoTunnelManager(config *Config, metricsSetup *MetricsConfig) (Tunnel,
 		OriginCert:        config.OriginCert, // []byte{}
 		TlsConfig:         tlsConfig,
 		ClientTlsConfig:   httpTransport.TLSClientConfig, // *tls.Config
-		Retries:           5,
+		Retries:           8,                             // backoff retry maximum=2^8 seconds
```

Comment:
we should expose these items as annotations on the ingress definition (the change to the default from 5 to 8 is fine, I like the comment, more of a moving forward plan)